rails/app/views/themes/ngss-assessment/shared/_footer.html.haml

```diff
@@ -14,20 +14,27 @@
         = succeed ". " do
           = link_to 'Concord Consortium', 'http://concord.org/', :target => '_blank'
         Past partnership included work with SRI International.
-      %p.ngsa-partners
-        Additional partners include 
-        = succeed ": " do
-          = link_to 'UChicago STEM Education', 'https://stemed.uchicago.edu/', :target => '_blank'
-        %img{:src => "https://ngss-assessment-resources.concord.org/uc-stem-ed.png", :style => "height: auto; width: 160px;"}
+      %p.ngsa-partners{:style => "margin-top: 20px"}
+        Additional partners include
+        = link_to 'UChicago STEM Education', 'https://stemed.uchicago.edu/', :target => '_blank'
+        and
+        = succeed "." do
+          = link_to 'The Lawrence Hall of Science', 'https://lawrencehallofscience.org/', :target => '_blank'
+      %p.ngsa-partners-images
+        %img{:src => "https://ngss-assessment-resources.concord.org/uc-stem-ed.png", :style => "height: 100%; width: 160px;"}
+        %img{:src => "https://ngss-assessment-resources.concord.org/Lawrence-Hall-of-Science-logo.inverted.png", :style => "height: 100%; width: 160px;"}
```

Comment:
You probably already considered this and there's a good reason not to, but you might want to move the inline style rules here to the stylesheet. Not required, though, of course.